Ciphers/AffineCipher.js

```diff
@@ -28,6 +28,27 @@ function inverseMod (a, m) {
   }
 }
 
+/**
+ * GetEuclidGCD Euclidean algorithm to determine the GCD of two numbers
```

Comment:
Please don't copy this but rather import the GCD func.

Reply:
Sorry initally when i import the js file, when debug it keep giving me error module not found, but during test it pass i not sure whether it correct or not...